app/views/comments/_form.html.erb

```diff
@@ -0,0 +1,13 @@
+<%= form_with model: [@article, @comment] do |form| %>
```

Comment:
when doing partial form it can access without `@` before variable because we pass by something like this
```
<%= render "form", article: @article %>
```